doc/install/installers.rst

```diff
@@ -74,12 +74,29 @@ Platform-specific installers
             :ref-type: ref
             :color: primary
             :shadow:
-            :class: font-weight-bold mt-3
+            :class: font-weight-bold mt-3 install-download-button
 
             |cloud-arrow-down| |ensp| Download for Windows
 
         **Supported platforms:** Windows 10 and newer
 
+.. card::
+    :class-body: text-center
+    :class-card: install-download-alert hidden
+
+    .. We have to use a button-link here because button-ref doesn't properly nested parse the inline code
+
+    .. button-link:: ./ides.html
+        :ref-type: ref
+        :color: success
+        :shadow:
+        :class: font-weight-bold mt-3
+        :click-parent:
+
+        |code| Go to IDE Setup
```

Comment:
the code icon seems weird to me here. Maybe:

```suggestion
        |rocket| Go to IDE Setup
```
or perhaps `|terminal|` ?

Reply:
(nitpick, feel free to ignore)

Reply:
Those were the three I debated between. I went with `|code|` because it matches what we use below, but I think I'll switch it to `|rocket|` here because it's more of a "getting started / get going" sort of thing in this context, and I agree the code looks weird to start a button